end-to-end-test/remote/specs/core/groupComparisonLollipop.spec.js

```diff
@@ -0,0 +1,234 @@
+    .goToUrlAndSetLocalStorage;
+var waitForNetworkQuiet = require('../../../shared/specUtils')
+    .waitForNetworkQuiet;
+var setInputText = require('../../../shared/specUtils').setInputText;
+var setSettingsMenuOpen = require('../../../shared/specUtils')
+    .setSettingsMenuOpen;
+const { getElementByTestHandle } = require('../../../shared/specUtils');
+
+const CBIOPORTAL_URL = process.env.CBIOPORTAL_URL.replace(/\/$/, '');
+
+describe('group comparison mutations tab tests', function() {
+    describe('lollipop alerts and plot display', function() {
+        before(function() {
+            goToUrlAndSetLocalStorage(
+                `${CBIOPORTAL_URL}/comparison?comparisonId=634006c24dd45f2bc4c3d4aa`
+            );
+            $('div[data-test="ComparisonPageOverlapTabDiv"]').waitForExist();
+        });
+
+        it('too many groups alert displayed when more than 2 groups selected', function() {
+            $('a.tabAnchor_mutations').waitForExist();
+            $('a.tabAnchor_mutations').click();
+            browser.pause(1000);
+            assert.equal(
+                $('[data-test="TooManyGroupsAlert"]').isDisplayed(),
+                true
+            );
+        });
+
+        it('not enough groups alert displayed when less than 2 groups selected', function() {
+            $('a=Deselect all').waitForExist();
+            $('a=Deselect all').click();
+            browser.pause(1000);
+            assert.equal(
+                $('[data-test="NotEnoughGroupsAlert"]').isDisplayed(),
+                true
+            );
+            $(
+                '[data-test="groupSelectorButtonColon Adenocarcinoma"]'
+            ).waitForExist();
+            $('[data-test="groupSelectorButtonColon Adenocarcinoma"]').click();
+            browser.pause(1000);
+            assert.equal(
+                $('[data-test="NotEnoughGroupsAlert"]').isDisplayed(),
+                true
+            );
+        });
+
+        it('lollipop plot displayed when 2 groups selected', function() {
+            $(
+                '[data-test="groupSelectorButtonColorectal Adenocarcinoma"]'
+            ).waitForExist();
+            $(
+                '[data-test="groupSelectorButtonColorectal Adenocarcinoma"]'
+            ).click();
+            $('[data-test="LollipopPlot"]').waitForExist({ timeout: 20000 });
+            assert.equal($('[data-test="LollipopPlot"]').isDisplayed(), true);
+        });
+    });
+
+    describe('lollipop tooltip display', function() {
+        it('displays double tooltip when lollipop is present in both plots at the same position', function() {
+            $('.lollipop-0').waitForExist();
+            $('.lollipop-0').moveTo();
+            $('[data-test="tooltip-1450-Colon Adenocarcinoma"]').waitForExist();
+            $(
+                '[data-test="tooltip-1450-Colorectal Adenocarcinoma"]'
+            ).waitForExist();
+            assert.equal(
```

Comment:
instead of these big assertions, we can just do 
```$('[data-test="tooltip-1450-Colon Adenocarcinoma"]').waitForExist();```
which as explained above has the same effect as an assertion.   i know if feels weird not do an assertion in a test, but it's ok.